blocks/init/src/Blocks/components/icon/manifest.json

```diff
@@ -0,0 +1,113 @@
+		}
+	},
+	"attributes": {
+		"iconColor": {
+			"type": "string",
+			"default": "primary"
+		},
+		"iconName": {
+			"type": "string",
+			"default": "play-icon"
+		},
+		"iconUse": {
+			"type": "boolean",
+			"default": false
+		}
+	},
+	"options": {
+		"icons": [
+			{
+				"label": "Map pin icon",
+				"value": "map-pin-icon"
+			},
+			{
+				"label": "Contact icon",
+				"value": "contact-icon"
+			},
+			{
+				"label": "Calendar icon",
+				"value": "calendar-icon"
+			},
+			{
+				"label": "Download icon",
+				"value": "download-icon"
+			},
+			{
+				"label": "Share icon",
+				"value": "share-icon"
+			},
+			{
+				"label": "Play icon",
+				"value": "play-icon"
+			},
+			{
+				"label": "Arrow up icon",
+				"value": "arrow-up-icon"
+			},
+			{
+				"label": "Arrow right icon",
+				"value": "arrow-right-icon"
+			},
+			{
+				"label": "Chevron left icon",
+				"value": "chevron-left-icon"
+			},
+			{
+				"label": "Chevron right icon",
+				"value": "chevron-right-icon"
+			},
+			{
+				"label": "Twitter icon",
+				"value": "twitter-icon"
+			},
+			{
+				"label": "Facebook icon",
+				"value": "facebook-icon"
+			},
+			{
+				"label": "Linkedin icon",
+				"value": "linkedin-icon"
+			},
+			{
+				"label": "Close icon",
+				"value": "close-icon"
+			},
+			{
+				"label": "Rounded rectangle icon",
+				"value": "rounded-rectangle-icon"
+			}
+		],
+		"colors": [
+			"primary",
+			"black",
+			"light",
+			"white"
+		]
+	},
+	"icons": {
+		"map-pin-icon": "<svg width='24' height='24' viewBox='0 0 24 24' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M4 10C4 5.58172 7.58172 2 12 2C16.4183 2 20 5.58172 20 10C20 13.0981 17.9843 16.1042 15.774 18.4373C14.6894 19.5822 13.6013 20.5195 12.7833 21.1708C12.4789 21.4133 12.213 21.6152 12 21.7726C11.787 21.6152 11.5211 21.4133 11.2167 21.1708C10.3987 20.5195 9.31061 19.5822 8.22595 18.4373C6.01574 16.1042 4 13.0981 4 10ZM11.4448 23.8317C11.445 23.8319 11.4453 23.8321 12 23L11.4453 23.8321C11.7812 24.056 12.2188 24.056 12.5547 23.8321L12 23C12.5547 23.8321 12.555 23.8319 12.5552 23.8317L12.556 23.8312L12.5581 23.8298L12.5648 23.8253L12.5877 23.8098C12.6072 23.7966 12.6349 23.7776 12.6704 23.753C12.7415 23.7038 12.8435 23.6321 12.9722 23.5392C13.2295 23.3534 13.5936 23.0822 14.0292 22.7354C14.8987 22.043 16.0606 21.0428 17.226 19.8127C19.5157 17.3958 22 13.9019 22 10C22 4.47715 17.5228 0 12 0C6.47715 0 2 4.47715 2 10C2 13.9019 4.48426 17.3958 6.77405 19.8127C7.93939 21.0428 9.10133 22.043 9.97082 22.7354C10.4064 23.0822 10.7705 23.3534 11.0278 23.5392C11.1565 23.6321 11.2585 23.7038 11.3296 23.753C11.3651 23.7776 11.3928 23.7966 11.4123 23.8098L11.4352 23.8253L11.4419 23.8298L11.444 23.8312L11.4448 23.8317ZM12 6C9.79086 6 8 7.79086 8 10C8 12.2091 9.79086 14 12 14C14.2091 14 16 12.2091 16 10C16 7.79086 14.2091 6 12 6ZM10 10C10 8.89543 10.8954 8 12 8C13.1046 8 14 8.89543 14 10C14 11.1046 13.1046 12 12 12C10.8954 12 10 11.1046 10 10Z' fill='currentColor' /></svg>",
```

Comment:
Do we need to have the `-icon` suffix?  Seems a bit redundant.

Reply:
agreed